### DISCUSSION TITLE
计算机科学导论 | HackWay

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#14] [Announcements]
# 计算机科学导论 | HackWay

计算机专业入门的第一课通常被称之为“CS0”，课程名字为 “计算机科学导论” 或 “计算机科学概论”，这门课主要向没有学习过任何前置知识的新人介绍计算机科学是什么，当走进大学时应该学习什么，非常适合高考刚毕业的准大学生、大一新生、计算机自学爱好者等。

http://localhost:3000/docs/cs/freshman/csintro

**Comment by lidongyx** [comment]


**Reply by lidongyx** [reply]
好棒的样子哦
